lib/src/connector/ischool_plus_connector.dart

```diff
@@ -58,13 +66,27 @@ class ISchoolPlusConnector {
         final value = node.attributes['value'];
         data[name] = value;
       }
+
+      // Step 1
       final jumpUrl = tagNode.getElementsByTagName("form")[0].attributes["action"];
-      parameter = ConnectorParameter(jumpUrl);
+      parameter = ConnectorParameter("${NTUTConnector.host}$jumpUrl");
       parameter.data = data;
 
       Response<dynamic> jumpResult = (await Connector.getDataByPostResponse(parameter));
+      tagNode = html.parse(jumpResult.data.toString().trim());
+      nodes = tagNode.getElementsByTagName("a");
+      final redirectUrl = nodes[0].attributes["href"];
+
+      // Step 2
+      parameter = ConnectorParameter(redirectUrl);
+      await Connector.getDataByGet(parameter);
+
       // Perform retry for cryptic API errors (?).
       // If the string `connect lost` be found in the response, we will do the retry.
+
+      // [2023-10-21] We may don't need this since the step is changed.
+      // Clean on next PR.
+
```

Comment:
```suggestion
      // TODO: remove the ischool retry loop since ....
```

Suggest changing this comment to a TODO comment.

Reply:
Fix on 2d99063 and 5c51539